src/Loggable/Entity/Repository/LogEntryRepository.php

```diff
@@ -107,7 +107,7 @@ public function revert($entity, $version = 1)
                         }
                     }
                 }
-                $filled = 0 === count($fields);
+                $filled = 0 === (is_array($fields) || $fields instanceof \Countable ? count($fields) : 0);
```

Comment:
`$fields` is a non-typed element of the `$config` array: `$fields = $config['versioned'];`. How should we handle it?

Reply:
We could just do `assert(is_array($fields));` before the call. Or maybe add phpdoc like:

```php
/** @var array $fields **/
$fields = $config['versioned'];
```
It would be nice to defined somewhere an array shape for the configuration like: 

```php
/** @return array{versioned: array, ...} */
```
But just realised that `getConfiguration()` method is common for the extensions